### DISCUSSION TITLE
Explainability on Link Prediction in Heterogeneous Graph

### DISCUSSION THREAD

**Discussion by claudiostanzione** [pyg-team/pytorch_geometric#7720] [Q&A]
Hello everyone, I am new to working with graphs and currently conducting research that focuses on applying explainability to link prediction in a heterogeneous graph. Although I have come across several examples, none of them are suitable for my specific case. The result I need to explain is a binary classification rather than a regression, for instance. I would greatly appreciate any advice on how to structure the model to incorporate explainability, or any recommendations for existing notebooks that tackle a similar problem with the same structure I have described. Thank you in advance for your  help.

**Comment by rusty1s** [comment]
Is this issue resolved?
